unthink-stack/src/server/config/config.ts

```diff
@@ -23,3 +24,6 @@ export const nunjucksFatalErrorTemplate = getEnvironmentValue('NUNJUCKS_TEMPLATE
 export const nunjucksUnauthorizedTemplate = getEnvironmentValue('NUNJUCKS_TEMPLATE_UNAUTHORIZED');
 
 export const contentBasePath: string = getEnvironmentValue('CONTENT_BASE_PATH');
+
+export const appName: string = JSON.parse(fs.readFileSync('./package.json').toString()).name;
+export const appVersion: string = JSON.parse(fs.readFileSync('./package.json').toString()).version;
```

Comment:
No need to read and parse this twice. Read the package into a variable then you can use it.

```
const package = JSON.parse(fs.readFileSync('./package.json').toString());
```

Or something like that and then use `package.name` and `package.template`

Otherwise looks fine.

Reply:
Fan of destructuring like this or no?
```
export const { name: appName, version: appVersion } = JSON.parse(fs.readFileSync('./package.json').toString());
```

Reply:
Destructuring, yes. In this instance, no. IMO it makes it harder to read/understand the config file at a glance. Config files generally follow a one-item-per-line scheme.